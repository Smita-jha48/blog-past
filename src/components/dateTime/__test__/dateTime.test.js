import { render } from '@testing-library/react';
import DateTime from '../dateTime';

describe('DateTime', () => {
  it('should render correctly', () => {
    const mockData = {
      date: '8.9.23',
      readingTime: '2 mins',
    };
    const { asFragment } = render(<DateTime {...mockData} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
