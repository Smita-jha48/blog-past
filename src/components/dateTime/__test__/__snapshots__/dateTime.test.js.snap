// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateTime should render correctly 1`] = `
<DocumentFragment>
  <div
    class="upper"
  >
    <p>
      8.9.23
    </p>
    <p>
      2 mins
    </p>
  </div>
</DocumentFragment>
`;
